pkg/controller/handlers/scheduledauditlogexport/scheduledauditlogexport.go

```diff
@@ -0,0 +1,128 @@
+package scheduledauditlogexport
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/adhocore/gronx"
+	"github.com/obot-platform/nah/pkg/router"
+	v1 "github.com/obot-platform/obot/pkg/storage/apis/obot.obot.ai/v1"
+	"github.com/obot-platform/obot/pkg/system"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type Handler struct {
+}
+
+func NewHandler() *Handler {
+	return &Handler{}
+}
+
+func (h *Handler) ScheduleExports(req router.Request, resp router.Response) error {
+	scheduledExport := req.Object.(*v1.ScheduledAuditLogExport)
+
+	if !scheduledExport.Spec.Enabled {
+		return nil
+	}
+
+	next, err := calculateNextRunTime(scheduledExport)
+	if err != nil {
+		return fmt.Errorf("failed to calculate next run time: %w", err)
+	}
+
+	if until := time.Until(next); until > 0 {
+		if until < 10*time.Hour {
+			resp.RetryAfter(until)
+		}
+		return nil
+	}
+
+	if err := h.createExportFromSchedule(req, scheduledExport, next); err != nil {
+		return err
+	}
+
+	scheduledExport.Status.LastRunAt = &[]metav1.Time{metav1.Now()}[0]
+
+	return req.Client.Update(req.Ctx, scheduledExport)
```

Comment:
```suggestion
	return req.Client.Status().Update(req.Ctx, scheduledExport)
```

Again, not totally sure if this is needed.